WooCommerce/src/main/kotlin/com/woocommerce/android/extensions/NumberExt.kt

```diff
@@ -33,4 +33,8 @@ infix fun <T> Comparable<T>?.greaterThan(other: T) =
     this?.let { it > other }
         ?: false
 
+infix fun <T> Comparable<T>?.lesserThan(other: T) =
```

Comment:
```suggestion
infix fun <T> Comparable<T>?.smallerThan(other: T) =
```
I'd opt for renaming it to `smallerThan`.

Reply:
Yeah, I am not an English expert but I ran "what is opposite to greater" in google and chatgpt and the result was that, so I guess this is grammatically correct =)

<img width="450" alt="image" src="https://user-images.githubusercontent.com/4923871/229441701-2e815903-4183-4103-9631-01cb8b7d5ba2.png">